src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -140,7 +140,13 @@ class JesBackend extends Backend with LazyLogging {
    * Delete authentication file in gcs once workflow is in a terminal state.
    */
   override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = {
-    Future(JesConnection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow))))
+    // TODO have this retry and/or check for the file's existence explicitly
+    Try(JesConnection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow)))) match {
+      case Failure(f) => logger.warn(s"Could not delete GCS file ${gcsAuthFilePath(workflow)}: ${f.getMessage}")
+      case Success(_) =>
+    }
```

Comment:
I don't know what you mean by 'recover'.  Also, this block I added really add a lot of value as it is.  `f.getMessage` still prints out the ugly JSON blob with the HTTP error code in it.  The only thing my code added was a "warn" log level to the message.

Also I couldn't figure out, using Google's Java API, how to ask if a GCS path exists or not...  that's what I really wanted to do, and then if it does exist, put the `deleteObject` call in a retry loop